index.html

```diff
@@ -3,7 +3,435 @@
+                contribute to open source.
+              </p>
+              <div class="comment-reply">
+                <i class="fas fa-thumbs-up" aria-hidden="true"></i>
+                <span class="thousands">8K</span>
+                <span class="reply">REPLY</span>
+              </div>
+              <div class="comment-replies">
+                <i class="fas fa-sort-down" aria-hidden="true"></i>
+                <p>View 5 replies</p>
+              </div>
+            </div>
+          </div>
+
+          <div class="other-comment">
+            <img class="comment-image" src="./image/commentyoutube.jpg" alt="Avatar image" />
+            <div class="comment-block">
+              <h3>Adeyemi <span> 4 year ago</span></h3>
+              <p>
+                Thumbs up!..really interesting topic,am an aspiring web developer that want to
+                contribute to open source.
+              </p>
+              <div class="comment-reply">
+                <i class="fas fa-thumbs-up" aria-hidden="true"></i>
+                <span class="thousands">8K</span>
+                <span class="reply">REPLY</span>
+              </div>
+              <div class="comment-replies">
+                <i class="fas fa-sort-down" aria-hidden="true"></i>
+                <p>View 5 replies</p>
+              </div>
+            </div>
+          </div>
+
+          <div class="other-comment">
+            <img class="comment-image" src="./image/comment2youtube.jpg" alt="Avatar image" />
+            <div class="comment-block">
+              <h3>Adeyemi <span> 4 year ago</span></h3>
+              <p>
+                Thumbs up!..really interesting topic,am an aspiring web developer that want to
+                contribute to open source.
+              </p>
+              <div class="comment-reply">
+                <i class="fas fa-thumbs-up" aria-hidden="true"></i>
+                <span class="thousands">8K</span>
+                <span class="reply">REPLY</span>
+              </div>
+              <div class="comment-replies">
+                <i class="fas fa-sort-down" aria-hidden="true"></i>
+                <p>View 5 replies</p>
+              </div>
+            </div>
+          </div>
+        </section>
+      </section>
+      <aside class="aside-container">
+        <div class="play">
+          <p class="up-next">Up next</p>
+          <p class="autoplay">AUTOPLAY</p>
+        </div>
+
+        <div class="card">
+          <div class="img-container">
+            <img class="aside-img" alt="SJ Tech" src="./image/asideyoutube.jpg" />
+          </div>
+          <div class="text-container">
+            <h3 class="h3">
+              'How to Get a Job at the Big 4 - Amazon, Facebook, Google &amp; Microsoft' by Sean Lee
+            </h3>
+            <p class="views">SJ Tech</p>
+            <p class="views">3.2M views<span> • </span>3 years ago</p>
+          </div>
+        </div>
```

Comment:
I think we can make the `.card` element a `figure` element instead of a `div` element where the `figcaption` can be the `.text-container` element.

Learn more about [`<figure>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure) here.